artifactory/commands/buildinfo/publish.go

```diff
@@ -91,6 +93,35 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfo.Started)
+	if errorutils.CheckError(err) != nil {
+		return err
+	}
+
+	// Convert from nanoseconds to milliseconds
```

Comment:
Let's move all the code that builds the build-info URL into a separate private method of this struct, and then add a unit test for it (in publish_test.go). The unit test can then check that the URL is build properly  when the method receieved different versions.